script/main.py

```diff
@@ -213,9 +213,9 @@ def insert_users_to_db(conn, people):
       nat
     ]
 
-    people.append(columns)      
+    users.append(columns)      
     
-  insert_users_to_table(conn, people)
+  insert_users_to_table(conn, users)
 
 
 def init_db(conn):  
```

Comment:
Does it work?

Reply:
Please provide short function names

Reply:
yes it work `python script/main.py init`